pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[tool.poetry]
```

Comment:
For flake8 configuration we are using `flake8-pyproject` so we can centralize configurations inside pyproject.toml. Please take a look to https://github.com/cloudblue/connect-eaas-core/blob/master/pyproject.toml.